viadot/flows/prefect_logs.py

```diff
@@ -1,3 +1,4 @@
+from sqlite3 import Timestamp
```

Comment:
Is it a good import? Timestamp from sqlite3?

Reply:
changed to 
from pandas import Timestamp